src/interface.jl

```diff
@@ -176,10 +176,17 @@ runtime_module(@nospecialize(job::CompilerJob)) = error("Not implemented")
 isintrinsic(@nospecialize(job::CompilerJob), fn::String) = false
 
 # provide a specific interpreter to use.
+if isdefined(Base, :method_instance)
```

Comment:
```suggestion
if VERSION >= v"1.11.0-DEV.1552"
```

I personally prefer a version-based check, especially here where `Base.method_instance` is not related to the `token` kwarg introduced here.